moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1231,7 +1231,7 @@ bool RobotState::getJacobian(const JointModelGroup* group, const LinkModel* link
     const JointModel* pjm = link->getParentJointModel();
     if (pjm->getVariableCount() > 0)
     {
-      if (not group->hasJointModel(pjm->getName()))
+      if (!group->hasJointModel(pjm->getName()))
```

Comment:
what a shame MSVC is still not C++ standards compliant. wasn't there a compiler flag to enable a more compliant mode?